.rubocop_todo.yml

```diff
@@ -365,43 +364,6 @@ Rails/ApplicationController:
   Exclude:
     - 'app/controllers/health_controller.rb'
 
-# Configuration parameters: Database, Include.
-# SupportedDatabases: mysql, postgresql
-# Include: db/**/*.rb
-Rails/BulkChangeTable:
```

Comment:
I believe this whole block is no longer considered eligible for review as side effect of the read replica config changes. Part of the cop is detecting a database key in the yaml which no longer exists. Arguably a bug in the cop, but I think we are not going to fix old migrations anyway.